translations/de/info/task_description.html

```diff
@@ -0,0 +1,119 @@
+    wenig vorwärts und ein gutes Stück zurück rannte und sich heiser bellte, bis er
+    sich zuletzt mit zum Munde heraushängender Zunge und halb geschlossenen Augen, 
+    ganz außer Athem hinsetzte.
+</p>
+
+    <p style="float: right">
+        "Alice's Abenteuer im Wunderland" Lewis Carroll<br>
+        Aus dem Englischen von Antonie Zimmermann, 1869
+    </p>
+
+    <p style="clear: both;"></p>
+
+    <p style="text-indent: 0; text-align: center;">
+        Mein Fund, ein Stock, er wog 2 Pfund:<br>
+        Eines Tages, ich zersägte ihn<br>
+        Acht Stück von gleich Gewicht!<br>
+        Wieviel wiegte ein jedes noch?<br>
+        (Jeder der sagt, "ein Viertelpfund", liegt falsch.)
+    </p>
+
+    <p style="float: right">
+        "Alice im Wunderland der Rätsel ." Lewis Carroll <br>
+        Freie Übersetzung
+    </p>
+
+</div>
+<p>
+    Die Roboter wollen den Stock in mehrere Stücke sägen. Die Länge des Stocks ist <strong>N</strong> Zoll.
+    Wir sollten unseren Robotern dabei helfen. Jedes Stück muss eine ganzzahlige Länge aufweisen (1, 2, 3 .. Zoll, 
+    nicht 1.2).
+    Wir lieben Zahlenreihen und speziell die Dreieckszahlen (<a href="https://de.wikipedia.org/wiki/Dreieckszahl">mehr 
+    über Dreieckszahlen auf Wikipedia</a>), also sollst Du den Stock so zersägen, dass die resultierenden Längen eine 
+    fortlaufende Serie der Dreieckszahlen bilden. Wähle immer die maximale Größe (Länge der Reihe).
+    Die Teile müssen alle verschiedene Längen haben (keine Wiederholung).
+    Zum Beispiel: <strong>64</strong> soll in <strong>15, 21, 28</strong> geteilt werden,
+    da <strong>28, 36</strong> kürzer ist, und <strong>1, 3, 15, 45</strong> keine fortlaufende Reihe bildet.
+</p>
+
+<p>
+    Gegeben ist die Länge des Stocks (N).
+    Deine Lösung soll eine Liste mit Längen der Teile in aufsteigender Folge zurückgeben.
+    Wenn es nicht möglich ist, also das Problem keine Lösung hat, dann gib eine leere Liste zurück.
+</p>
+
+<p style="text-align: center;">
+    <img class="for_info_only" title="sawing" src="{{MEDIA}}sawing.png" alt="sawing" width="440"/>
+    <img class="for_editor_only" title="sawing" src="{{MEDIA}}sawing.png" alt="sawing" width="380px"/>
+</p>
+
+<p>
+    <strong>Eingabe: </strong> Die Länge des Stockes als Ganzzahl.
+</p>
+
+<p>
+    <strong>Ausgabe: </strong> Ein Ausschnitt der Dreieckszahlen als Liste von Ganzzahlen (sortiert in 
+    absteigender Folge) oder eine leere Liste.  
+</p>
+
+
+<div class="for_info_only">
+    <p>
+        <strong>Beispiel:</strong>
+    </p>
+    <pre class="brush: python">
+checkio(64) == [15, 21, 28]
+checkio(371) == [36, 45, 55, 66, 78, 91]
+checkio(225) == [105, 120]
+checkio(882) == []
+    </pre>
+</div>
+
+<p class="for_info_only">
+    <strong>Wozu das gut ist: </strong>
+    In dieser Aufgabe lernst Du etwas über Dreieckszahlen.
+    Eine Dreieckszahl ist gleich der Anzahl Teile um ein gleichseitiges Dreieck zu legen.
```

Comment:
No, I don't want to repeat the obvious from wikipedia.
The original points explicitly the idea to form an equilateral triangle. This is also found in the wiki article, but why shouldn't I stay with the idea from the orignal?